src/components/clusters/wizards/rosa/ClusterSettings/Details/Details.tsx

```diff
@@ -559,7 +559,7 @@ function Details() {
                     hint={
                       <>
                         {constants.enableUserWorkloadMonitoringHelp}
-                        <ExternalLink href={links.ROSA_MONITORING}>Learn more</ExternalLink>
+                        <ExternalLink href={links.ROSA_CLASSIC_MONITORING}>Learn more</ExternalLink>
```

Comment:
why is the HCP link not used here?

i assume the answer to [that comment][1] will address this one, too.


[1]: https://github.com/RedHatInsights/uhc-portal/pull/280/changes#r2608127611